### DISCUSSION TITLE
Alternative color schemes for alpine coaster

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#21774] [Feature requests] [answered]
Not sure why the alpine coaster is the only tracked ride that doesn’t allow alternative color schemes. I can report this as a bug if it was not intentional

**Comment by Gymnasiast** [comment]
Let’s ask @X123M3-256 , then.

**Reply by X123M3-256** [reply]
I don't know. I am guessing there is a flag that is needed to enable this feature that I forgot to include.

I think the alpine coaster is (intentionally) missing some flags that coaster types almost always have, like the one that makes it calculate G forces. IIRC there is a standard macro that adds all of these flags at once, which the alpine coaster doesn't use, I think I might have left out a flag that should have been enabled. That would be my guess, I'd have to look into the code. I hadn't seen this bug reported before.

**Reply by Gymnasiast** [reply]
That’s OK, I just wanted to check if it was intentional.

**Comment by Gymnasiast** [comment, marked as answer]
Fixed by https://github.com/OpenRCT2/OpenRCT2/commit/d53b6922a348f7523a2d6b9b59fff52a92c56b70.
